apps/root/src/pages/earn/components/strategies-table/components/columns/index.tsx

```diff
@@ -20,25 +32,93 @@ export enum StrategyColumnKeys {
   GUARDIAN = 'guardian',
   SAFETY = 'safety',
+  WALLET = 'wallet',
+  TOTAL_INVESTED = 'totalInvested',
+  CURRENT_PROFIT = 'currentProfit',
 }
 
-const StyledYieldTypeBox = styled.div`
+const StyledWalletsPlusIndicator = styled(ContainerBox)`
+  position: absolute;
+  ${({ theme: { palette, spacing } }) => `
+    padding: ${spacing(1)};
+    right: -${spacing(2.5)};
+    border-radius: 50%;
+    border: 1px solid ${colors[palette.mode].border.border2};
+    background: ${colors[palette.mode].background.tertiary};
+  `}
+`;
+
+const StyledBoxedLabel = styled.div`
   ${({ theme: { palette, spacing } }) => `
   padding: ${spacing(1)} ${spacing(3)};
   border-radius: ${spacing(2)};
-  border: 1px solid ${colors[palette.mode].violet.violet300};
+  border: 1px solid ${colors[palette.mode].border.border2};
+  background: ${colors[palette.mode].background.tertiary};
   `}
 `;
 
-export interface StrategyColumnConfig {
+const CustomTableCell = styled(TableCell)`
+  ${({ theme: { spacing } }) => `
+    padding-top: ${spacing(0.5)};
+    padding-bottom: ${spacing(0.5)};
+  `}
+`;
+
+const StyledBoxedOwners = ({ owners }: { owners: ViemAddress[] }) => {
+  const mode = useThemeMode();
+
+  return owners.length >= 2 ? (
+    <CustomTableCell>
+      <Tooltip
+        title={
+          <ContainerBox flexDirection="column" gap={0.5}>
+            {owners.map((owner) => (
+              <Typography key={owner} variant="bodySmallRegular">
+                <Address address={owner} trimAddress />
+              </Typography>
+            ))}
+          </ContainerBox>
+        }
+      >
+        <ContainerBox flexDirection="column" gap={0.5} style={{ position: 'relative' }} justifyContent="center">
+          {owners.slice(0, 2).map((owner) => (
+            <StyledBoxedLabel key={owner}>
+              <StyledBodySmallRegularTypo2>
+                <Address address={owner} trimAddress trimSize={3} />
+              </StyledBodySmallRegularTypo2>
+            </StyledBoxedLabel>
+          ))}
+          <StyledWalletsPlusIndicator>
+            <Typography variant="bodyExtraSmallBold" color={colors[mode].typography.typo2}>
+              +{owners.length - 2}
+            </Typography>
+          </StyledWalletsPlusIndicator>
+        </ContainerBox>
+      </Tooltip>
+    </CustomTableCell>
+  ) : (
+    <TableCell>
+      {owners.map((owner) => (
+        <StyledBoxedLabel key={owner}>
+          <StyledBodySmallRegularTypo2>
+            <Address address={owner} trimAddress trimSize={3} />
+          </StyledBodySmallRegularTypo2>
+        </StyledBoxedLabel>
+      ))}
+    </TableCell>
+  );
+};
+
+export interface StrategyColumnConfig<T extends StrategiesTableVariants> {
   key: StrategyColumnKeys;
   label: React.ReactNode;
-  renderCell: (data: TableStrategy) => React.ReactNode | string;
-  getOrderValue?: (data: TableStrategy) => string | number;
+  renderCell: (data: TableStrategy<T>) => React.ReactNode | string;
+  customCell?: boolean;
```

Comment:
custom would just mean its a reactNode that is being sent or why do we have this boolean?

Reply:
It was for adding a custom padding for bigger cells, but now it's gone :)